src/components/tooltip/tooltip-content.tsx

```diff
@@ -25,10 +25,30 @@ export class TooltipContent {
     @Prop({ reflect: true })
     helperLabel: string;
 
+    /**
+     * The maximum amount of characters before rendering 'label' and 'helperLabel' in two rows.
+     */
+    @Prop({ reflect: true })
+    maxlength: number;
+
+    private isLabelsTextLong = false;
+
+    public componentWillLoad() {
+        this.isLabelsTextLong =
+            this.label.length + this.helperLabel.length > this.maxlength;
```

Comment:
`helperLabel` is optional, so `this.helperLabel` might be undefined, in which case this line will throw an error, and the application will crash.

Reply:
The easiest and best solution is probably just to make `helperLabel` default to an empty string.